caustic/utils.py

```diff
@@ -416,3 +417,17 @@ def batch_lm(
         X = Xnew
         
     return X, L, C
+
+def gaussian(nx, ny, pixelscale, sigma, upsample = 1, dtype = torch.float32, device = None):
```

Comment:
I agree to match the signature of meshgrid with (pixelscale, nx, ny) instead of the other way around. 

I think if you already have a meshgrid then evaluating a gaussian is basically a one-liner so could be easily implemented by the user. I think the purpose of this gaussian function is just to do the meshgrid for you.